### DISCUSSION TITLE
i need help to login in

### DISCUSSION THREAD

**Discussion by Pearyman** [npm/feedback#727] [Registry]
hey team,  
long long ago i set the 2fa , but i forget the recovery code .
current situation is  i can't login in the account successfully.
because my device one-time verify can't use any more .

so when i login in , the 2fa verify needs device one-time code, but i don't hava the one -time code .
so i can't login in and make the 2fa disabled.

please help me , thank you .

my account is pearyman

**Comment by Pearyman** [comment]
and the support page is error, i can't submit
![image](https://user-images.githubusercontent.com/6932025/179143982-84191bbc-a4e9-47d9-939d-1d3e887ed5ff.png)

**Reply by MylesBorins** [reply]
Can you please try submitting a ticket again. I'm surfacing this to engineering
